### ISSUE TITLE
how do i run the repo locally?

### ISSUE THREAD

**Issue by owocki** [gitcoinco/grants-round#1]
would be great to get these instructions in the readme.

**Comment by owocki** [comment]
Graham | Gitcoin — Today at 8:15 AM
It looks like it should be: lerna bootstrap && yarn start 
(you will need lerna installed globally: yarn global add lerna)

**Comment by owocki** [comment]
hmmm i get an error when i do that

```
owocki@local /Users/owocki/Sites/grants-round~ % lerna bootstrap && yarn start                                                         (git)-[main] :
lerna notice cli v4.0.0
lerna info versioning independent
lerna info Bootstrapping 2 packages
lerna info Installing external dependencies
lerna ERR! npm install exited 1 in 'round-manager'
lerna ERR! npm install stderr:
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: round-manager@0.1.0
npm ERR! Found: react-scripts@5.0.1
npm ERR! node_modules/react-scripts
npm ERR!   react-scripts@"5.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react-scripts@"^4.0.0" from @craco/craco@6.4.3
npm ERR! node_modules/@craco/craco
npm ERR!   @craco/craco@"^6.4.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/owocki/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/owocki/.npm/_logs/2022-05-13T14_35_39_970Z-debug.log
lerna ERR! npm install exited 1 in 'round-manager'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
```

**Comment by owocki** [comment]
```
brew update
brew upgrade node
```
on mac this seems to help

**Comment by Cali93** [comment]
@thelostone-mc Do you think that it's better to go for a repo per package and publish some packages on npm if needed for code sharing or just go with this ?

> Lerna is dead... Long live Lerna!!! 😄 Thanks for Nrwl for taking the torch #3121
> 
> On a side note, I just released Lerna-Lite version [1.2.0](https://github.com/ghiscoding/lerna-lite/releases/tag/v1.2.0) which brings support for yarn/pnpm `workspace:` protocol. No need for alternatives anymore when you can use Lerna/Lerna-Lite 😉
> 
> Give it a try and happy coding 🎉
